web-components/src/components/cards/ProjectCard/ProjectCard.tsx

```diff
@@ -241,7 +241,7 @@ export function ProjectCard({
                       event.stopPropagation();
                       track &&
                         !isPrefinanceProject &&
-                        track<'buy1', Buy1Event>('buy1', {
```

Comment:
not directly related to this task, but here I'm just cleaning up the first generic param that was not really needed